src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/service/ReportService.kt

```diff
@@ -216,6 +217,36 @@ class ReportService(
     }.getOrNull()
   }
 
+  @Transactional
+  fun addDescriptionAddendum(id: UUID, descriptionAddendum: DescriptionAddendum): ReportWithDetails? {
+    return reportRepository.findOneEagerlyById(id)?.let { reportEntity ->
+      reportEntity.addDescriptionAddendum(
+        createdBy = descriptionAddendum.createdBy,
+        firstName = descriptionAddendum.firstName,
+        lastName = descriptionAddendum.lastName,
+        createdAt = descriptionAddendum.createdAt,
```

Comment:
Wouldn't this be `now` like below

Reply:
this happens in future pr https://github.com/ministryofjustice/hmpps-incident-reporting-api/pull/329/commits/0463a65c7944f21a66b8ae78d368944c9eebbf48